src/components/GraphFilter/GraphFilter.tsx

```diff
@@ -1,148 +1,74 @@
-    '2592000': { step: 86400, rateInterval: '1m' } // 30d - 2592000/86400=30 buckets which is 1 datapoint per 1d
-  };
-
-  let graphDuration: string = '600';
-  let graphStep: number = 100;
-  let graphRateInterval: string = '1m';
-  let graphLayout: any = DagreGraph.getLayout();
-  let graphNamespace: string = 'istio-system';
-
-  export const getGraphDuration = () => {
-    return graphDuration;
-  };
-
-  export const getGraphLayout = () => {
-    return graphLayout;
-  };
-
-  export const getGraphLayoutName = () => {
-    return graphLayout.name;
-  };
-
-  export const getGraphNamespace = () => {
-    return graphNamespace;
-  };
-
-  export const getGraphStep = () => {
-    return graphStep;
-  };
-
-  export const getGraphRateInterval = () => {
-    return graphRateInterval;
-  };
-
-  export const setGraphDuration = (value: string) => {
-    graphDuration = value;
-    graphStep = graphQueryOptionsPerDuration[value].step;
-    graphRateInterval = graphQueryOptionsPerDuration[value].rateInterval;
-  };
-
-  export const setGraphLayout = (value: string) => {
-    if (value === 'breadthfirst') {
-      graphLayout = BreadthFirstGraph.getLayout();
-    } else if (value === 'cola') {
-      graphLayout = ColaGraph.getLayout();
-    } else if (value === 'dagre') {
-      graphLayout = DagreGraph.getLayout();
-    } else if (value === 'cose-bilkent') {
-      graphLayout = CoseGraph.getLayout();
-    } else if (value === 'klay') {
-      graphLayout = KlayGraph.getLayout();
-    } else {
-      console.error('invalid graphLayout: ' + value);
-    }
-  };
-
-  export const setGraphNamespace = (value: string) => {
-    graphNamespace = value;
-  };
-}
 
 export class GraphFilter extends React.Component<GraphFilterProps, GraphFilterState> {
   constructor(props: GraphFilterProps) {
     super(props);
-
-    this.setNamespaces = this.setNamespaces.bind(this);
-    this.updateDuration = this.updateDuration.bind(this);
-    this.updateLayout = this.updateLayout.bind(this);
-    this.updateNamespace = this.updateNamespace.bind(this);
-
     this.state = {
-      graphDuration: GraphFilters.getGraphDuration(),
-      graphLayout: GraphFilters.getGraphLayout(),
-      graphNamespace: GraphFilters.getGraphNamespace(),
       availableNamespaces: []
     };
   }
 
   componentDidMount() {
+    // API.GetNamespaces() is also called in Services component.
+    // We should consolidate share the data somehow.
     API.GetNamespaces()
       .then(this.setNamespaces)
       .catch(error => {
         this.props.onError(error);
       });
   }
 
-  setNamespaces(response: any) {
+  setNamespaces = (response: any) => {
     this.setState({ availableNamespaces: response['data'] });
-  }
+  };
 
-  updateDuration(value: string) {
-    GraphFilters.setGraphDuration(value);
-    this.setState({ graphDuration: GraphFilters.getGraphDuration() });
-    this.props.onFilterChange();
-  }
+  updateInterval = (value: string) => {
```

Comment:
I know you mentioned this but for consistency you will have to immediately follow up with a PR for change to "Duration" terminology.